src/main/java/io/confluent/maven/resolver/ResolveKafkaVersionRangeMojo.java

```diff
@@ -0,0 +1,239 @@
+	private static final String CEVersionString = "-ce";
+	private static final String CCS = "ccs-kafka";
+	private static final String CCSVersionString = "-ccs";
+	private static final Pattern SNAPSHOT_TIMESTAMP = Pattern.compile("^(.*-)?([0-9]{8}\\.[0-9]{6}-[0-9]+)$");
+	private static final String CEKAFKAVERSION = "ce.kafka.version";
+	private static final String CCSKAFKAVERSION = "kafka.version";
+
+	/**
+	 * The group id of the artifact to resolve.
+	 */
+	@Parameter(property = "resolve.groupId", required = true)
+	private String groupId;
+
+	/**
+	 * The artifact id of the artifact to resolve.
+	 */
+	@Parameter(property = "resolve.artifactId", required = true)
+	private String artifactId;
+
+	/**
+	 * The version range of the artifact to resolve.
+	 */
+	@Parameter(property = "resolve.versionRange", required = true)
+	private String version;
+
+	/**
+	 * If <code>true</code>, the highest matching version is printed directly to the
+	 * console. This can be used with {@code mvn -q}.
+	 */
+	@Parameter(property = "resolve.printCE")
+	private boolean printCE;
+
+	@Parameter(property = "resolve.printCCS")
+	private boolean printCCS;
+
+	/**
+	 * Set to <code>true</code> to include snapshot versions in the resolution.
+	 */
+	@Parameter(property = "resolve.includeSnapshots", defaultValue = "false")
+	private boolean includeSnapshots;
+
+	/**
+	 * The name of the property that will be set to the highest matching version.
+	 */
+	@Parameter
+	private String property;
+
+	@Component
+	private RepositorySystem repoSystem;
+
+	@Parameter(defaultValue = "${repositorySystemSession}", readonly = true)
+	private RepositorySystemSession repoSession;
+
+	@Parameter(defaultValue = "${project.remoteProjectRepositories}", readonly = true)
+	protected List<RemoteRepository> remoteRepositories;
+
+	@Parameter(defaultValue = "${project}", readonly = true)
+	private MavenProject project;
+
+	@Override
+	public void execute() throws MojoExecutionException, MojoFailureException {
+		VersionRangeRequest request = new VersionRangeRequest();
+		request.setRepositories(remoteRepositories);
+		DefaultArtifact artifact = new DefaultArtifact(groupId, artifactId, null, version);
+		request.setArtifact(artifact);
+
+		String constraintText = artifact.toString() + ", " + (includeSnapshots ? "including" : "excluding")
+				+ " snapshots";
+		getLog().info("Resolving range for " + constraintText);
+		try {
+			VersionRangeResult CEResult = repoSystem.resolveVersionRange(repoSession, request);
+			VersionRangeResult CCSResult = repoSystem.resolveVersionRange(repoSession, request);
+			Version highestCEVersion = fetchHighestKafkaVersion(CE, CEResult, constraintText);
+			Version highestCCSVersion = fetchHighestKafkaVersion(CCS, CCSResult, constraintText);
+
+			getLog().info("Highest " + CE + " version: " + highestCEVersion);
+			getLog().info("Highest " + CCS + " version: " + highestCCSVersion);
+
+			if (printCE) {
+				System.out.println(highestCEVersion);
```

Comment:
This should probably go to the log, not directly to System.out

Reply:
This is system.out becuase it will be used as cmd style like 
```
 mvn io.confluent:resolver-maven-plugin:resolve-kafka-range \
 -Dresolve.groupId=org.apache.kafka -Dresolve.artifactId=kafka-client "-Dresolve.versionRange=[6.0.0-1, 6.0.1-1] -D printCCS -q" 
```